.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
There are so many Linter errors. You should apply  HTML & CSS best practices.

Reply:
Great work catching the Linter errors 👍, but I think it would be better to state the reason a change is requested and preferably point them to how to resolve the issue. The comment above could be written as follows: _Please be sure to address all linter errors as this will improve your code quality and consistency, you can find more information on HTML/CSS best practices [here](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/HTML)_